attributes.rst

```diff
@@ -0,0 +1,146 @@
+.. index::
+    single: Attributes
+
+Symfony Attributes Overview
+===========================
+
+Attributes are the successor of annotation since PHP 8. Attributes are native
+to the language and Symfony takes full advantage of them across the framework
+ans its different components.
+
+This article lists supported attributes of the framework, as well as their
+usages in the documentation.
```

Comment:
```suggestion
```

To make it more or less consistent with the introduction paragraphs in other reference articles.